astropy/io/votable/tests/converter_test.py

```diff
@@ -65,6 +66,42 @@ def test_unicode_mask():
     assert c.output("Foo", True) == ''
 
 
+def test_unicode_as_char():
+    config = {'verify': 'exception'}
+    #with catch_warnings(exceptions.W55) as w:
+    field = tree.Field(
+        None, name='unicode_in_char', datatype='char',
+        arraysize='*', config=config)
+    c = converters.get_converter(field, config=config)
+
+    c.parse("XXX")  # ASCII succeeds
+
+    with pytest.raises(
+        exceptions.W55,
```

Comment:
Easier to read if the continuation of `with` statement is at a different indentation level from the code inside the block.
```suggestion
            exceptions.W55,
```